draft-pignataro-eimpact-icmp.xml

```diff
@@ -458,7 +458,48 @@ The "Year" field must contain the year in which the certificate was issued, or 0
 
     <section title="Sample Use Case">
       <t>
-        This section explains a typical use case of the extensions.
+        The ICMP extension defined in this memo can be used to get sustainability metrics, either via a
+        "transactional" or a "scheduled automated" fashion, inside one administrative domain.
+        Mentioned in this section is a sample "transactional" use case of how the metrics will be
+        displayed to the user. As traceroute or ping utilities are based on ICMP, we will use
+        traceroute as an example.
+      </t>
+      <t>  
+        The flavour of traceroute that has been shown here, does support the ICMP extensions and is
+        capable of deliverintg the metrics in the extensions to the end user along with the nodes that 
+        the original datagram visited on it's way to the destination. And, it also is backwards-compatible,
+        i.e., it can also work well with those nodes which doesn't support ICMP extensions.
+      </t>
+      <t>
+        Host A executes a traceroute to 192.168.2.1 and the <xref target="use-case-1" /> shows how the user
```

Comment:
I do like the idea of saying "A user at host A" - it's just that "host A" is strange because it doesn't appear in the actual example, so why mention it?   How about: "A user at a host with IP address 192.168.120.126" ?  because that would somehow fit together with the example output - seeing all these IP addresses, it's likely that the user is in the same network.

Reply:
Valid point. I agree that "A user at a host with IP address 192.168.120.126 executes..." fits better with the example. I'll change that.

Reply:
Done.